src/commands/run.js

```diff
@@ -122,13 +132,18 @@ async function executeOnSas9(buildTarget, linesToExecute) {
   const sasjs = new SASjs({
     serverUrl: buildTarget.serverUrl,
     appLoc: buildTarget.appLoc,
-    serverType: buildTarget.serverType
+    serverType: buildTarget.serverType,
+    debug: true
   })
-  const executionResult = await sasjs.executeScriptSAS9(
-    linesToExecute,
-    serverName,
-    repositoryName
-  )
+  const executionResult = await sasjs
+    .executeScriptSAS9(linesToExecute, serverName, repositoryName)
+    .catch(async (err) => {
+      let log = err.payload.log
+
+      await createOutputFile(log)
+
+      throw new ErrorResponse('Find more error details in the log file.')
```

Comment:
It'd be good if we could output the path to the log file here.

Reply:
The path is printed when file is created. So in the terminal it will be above this print.